### DISCUSSION TITLE
How many historical records can I use?

### DISCUSSION THREAD

**Discussion by springer-exe** [PaddlePaddle/PaddleSpatial#25] [BaiduKDDCup2022] [answered]
The report say 

"For example, given at 6:00 A.M. today, it is required to eff ectively forecast the wind power generation beginning from 6:00 A.M. on this day to 5:50 AM on the day after tomorrow, **given a series of historical records** of the wind farm and the related wind turbines."

How many historical records can I use?

1 day? 2 days?

**Comment by ramber1836** [comment, marked as answer]
It is your own decision. You can take the baseline settings as an example.

All Best,
Xinjiang Lu

**Comment by ramber1836** [comment]
Hi, there,

The currently released data is dedicated to training and validating your
own model. Of course, you can use it to evaluate your model in the
current stage.
The next-released data for testing begins right after the currently
released data.

The procedure in the upcoming tests probably goes like this:
The setting of the prediction is length-Lx-Length-288, where Lx is the
input length for predicting the next 288 time points.
If the time span of the released test set is 60 days, then the last 30 days
will be evaluated for the overall performance. For instance, if one adopts
a rolling window with stride 1 on the test data, as well as the
length-288-length-288 prediction strategy, then, there will be around
60*24*6 - 288*2 + 1 predictions, and the last 30*24*6 predictions will be
examined to calculate the final score.

The evaluation script will be released in the next phase.

All The Best,
Xinjiang Lu

**Reply by havetry** [reply]
To be honest, I think the length of Lx should be fixed in advance, because for some models, this parameter is very important, such as TCN, we need to design the corresponding number of layers and void rate according to the input length.I hope the organizers will give this request a good thought.

**Reply by ramber1836** [reply]
We will take your concern into account. Thanks.
